webrtc.html

```diff
@@ -4306,6 +4307,10 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             <h2>Methods</h2>
             <dl data-link-for="RTCCertificate" data-dfn-for="RTCCertificate"
             class="methods">
+              <dt><dfn><code>getAlgorithms</code></dfn></dt>
+              <dd>
+                <p>Returns a sequence of recommended certificate algorithms.</p>
```

Comment:
Ah, I see, let me retract fingerprint and double down on my point that I think we need to be clear what it is :-)